parts/k8s/cloud-init/artifacts/sshd_config

```diff
@@ -1,19 +1,25 @@
+# Package generated configuration file
```

Comment:
let's get rid of these comments to save space (and in fact they are misleading :))